Basic/ClientDriven/Assets/Scripts/ServerIngredientPhysics.cs

```diff
@@ -14,13 +13,24 @@ public class ServerIngredientPhysics : NetworkBehaviour
 
     public override void OnNetworkObjectParentChanged(NetworkObject parentNetworkObject)
     {
-        SetPhysics(parentNetworkObject == null);
-    }
-
-    void SetPhysics(bool isEnabled)
-    {
-        m_Rigidbody.isKinematic = !isEnabled;
-        m_Rigidbody.interpolation = isEnabled ? RigidbodyInterpolation.Interpolate : RigidbodyInterpolation.None;
-        m_NetworkTransform.InLocalSpace = !isEnabled;
+        // The physic needs to be disabled when the ingredient is parented so that it's following the parent instead of moving freely.
+        if (parentNetworkObject != null)
```

Comment:
nitpick: in general, having a positive-phrased condition makes reading easier vs having a negative-phrased (I.E: "X is true" is easier to read than "X is not false"), so I'd swap the if and else branches.